2025-09-20T16:13:01.296+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
2025-09-20T16:13:01.300+0800	[31mERROR[0m	nacos_server/nacos_server.go:75	login has error Post "http://127.0.0.1:8080/nacos/v1/auth/users/login": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it.
2025-09-20T16:13:01.300+0800	[31mERROR[0m	nacos_server/nacos_server.go:199	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"goods-srv.json","group":"dev","tenant":"c7208266-4acb-431f-86e4-2c5f622aa1b1"}>, call domain error:<Get "http://127.0.0.1:8080/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=c7208266-4acb-431f-86e4-2c5f622aa1b1": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it.> , result:<>
2025-09-20T16:13:01.301+0800	[31mERROR[0m	nacos_server/nacos_server.go:199	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"goods-srv.json","group":"dev","tenant":"c7208266-4acb-431f-86e4-2c5f622aa1b1"}>, call domain error:<Get "http://127.0.0.1:8080/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=c7208266-4acb-431f-86e4-2c5f622aa1b1": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it.> , result:<>
2025-09-20T16:13:01.301+0800	[31mERROR[0m	nacos_server/nacos_server.go:199	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"goods-srv.json","group":"dev","tenant":"c7208266-4acb-431f-86e4-2c5f622aa1b1"}>, call domain error:<Get "http://127.0.0.1:8080/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=c7208266-4acb-431f-86e4-2c5f622aa1b1": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it.> , result:<>
2025-09-20T16:13:01.301+0800	[31mERROR[0m	config_client/config_client.go:202	get config from server error:Get "http://127.0.0.1:8080/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=c7208266-4acb-431f-86e4-2c5f622aa1b1": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
2025-09-20T16:13:01.304+0800	[31mERROR[0m	config_client/config_client.go:216	get config from cache  error:failed to read config cache file:tmp/nacos/cache\config\goods-srv.json@@dev@@c7208266-4acb-431f-86e4-2c5f622aa1b1,err:open tmp/nacos/cache\config\goods-srv.json@@dev@@c7208266-4acb-431f-86e4-2c5f622aa1b1: The system cannot find the path specified.  
2025-09-20T16:13:33.629+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
2025-09-20T16:18:03.821+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
2025-09-20T16:57:11.748+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
2025-09-20T16:58:57.906+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
2025-09-20T17:00:30.446+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
2025-09-20T18:16:41.044+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
2025-09-20T21:17:38.532+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
2025-09-20T21:17:43.533+0800	[31mERROR[0m	nacos_server/nacos_server.go:75	login has error Post "http://192.168.179.139:8848/nacos/v1/auth/users/login": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
2025-09-20T21:17:48.533+0800	[31mERROR[0m	nacos_server/nacos_server.go:199	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"goods-srv.json","group":"dev","tenant":"public"}>, call domain error:<Get "http://192.168.179.139:8848/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=public": context deadline exceeded (Client.Timeout exceeded while awaiting headers)> , result:<>
2025-09-20T21:17:53.534+0800	[31mERROR[0m	security/security_proxy.go:84	login has error Post "http://192.168.179.139:8848/nacos/v1/auth/users/login": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
2025-09-20T21:17:53.534+0800	[31mERROR[0m	nacos_server/nacos_server.go:199	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"goods-srv.json","group":"dev","tenant":"public"}>, call domain error:<Get "http://192.168.179.139:8848/nacos/v1/cs/configs?group=dev&tenant=public&dataId=goods-srv.json": context deadline exceeded (Client.Timeout exceeded while awaiting headers)> , result:<>
2025-09-20T21:17:58.535+0800	[31mERROR[0m	nacos_server/nacos_server.go:199	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"goods-srv.json","group":"dev","tenant":"public"}>, call domain error:<Get "http://192.168.179.139:8848/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=public": context deadline exceeded (Client.Timeout exceeded while awaiting headers)> , result:<>
2025-09-20T21:17:58.535+0800	[31mERROR[0m	config_client/config_client.go:202	get config from server error:Get "http://192.168.179.139:8848/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=public": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 
2025-09-20T21:18:03.535+0800	[31mERROR[0m	security/security_proxy.go:84	login has error Post "http://192.168.179.139:8848/nacos/v1/auth/users/login": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
2025-09-20T21:23:49.869+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
2025-09-20T21:23:51.903+0800	[31mERROR[0m	nacos_server/nacos_server.go:75	login has error Post "http://192.168.179.139:8848/nacos/v1/auth/users/login": dial tcp 192.168.179.139:8848: connectex: No connection could be made because the target machine actively refused it.
2025-09-20T21:23:53.955+0800	[31mERROR[0m	nacos_server/nacos_server.go:199	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"goods-srv.json","group":"dev","tenant":"public"}>, call domain error:<Get "http://192.168.179.139:8848/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=public": dial tcp 192.168.179.139:8848: connectex: No connection could be made because the target machine actively refused it.> , result:<>
2025-09-20T21:23:55.993+0800	[31mERROR[0m	nacos_server/nacos_server.go:199	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"goods-srv.json","group":"dev","tenant":"public"}>, call domain error:<Get "http://192.168.179.139:8848/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=public": dial tcp 192.168.179.139:8848: connectex: No connection could be made because the target machine actively refused it.> , result:<>
2025-09-20T21:23:58.038+0800	[31mERROR[0m	nacos_server/nacos_server.go:199	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"goods-srv.json","group":"dev","tenant":"public"}>, call domain error:<Get "http://192.168.179.139:8848/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=public": dial tcp 192.168.179.139:8848: connectex: No connection could be made because the target machine actively refused it.> , result:<>
2025-09-20T21:23:58.038+0800	[31mERROR[0m	config_client/config_client.go:202	get config from server error:Get "http://192.168.179.139:8848/nacos/v1/cs/configs?dataId=goods-srv.json&group=dev&tenant=public": dial tcp 192.168.179.139:8848: connectex: No connection could be made because the target machine actively refused it. 
2025-09-20T21:23:58.946+0800	[31mERROR[0m	security/security_proxy.go:84	login has error Post "http://192.168.179.139:8848/nacos/v1/auth/users/login": dial tcp 192.168.179.139:8848: connectex: No connection could be made because the target machine actively refused it.
2025-09-20T21:31:38.900+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
2025-09-20T21:42:22.537+0800	[34mINFO[0m	clients/client_factory.go:47	logDir:<tmp/nacos/log>   cacheDir:<tmp/nacos/cache>
